frontend/src/static/js/components/webstatus-overview-filters.ts

```diff
@@ -130,6 +135,14 @@ export class WebstatusOverviewFilters extends LitElement {
     ];
   }
 
+  constructor() {
+    super();
+    // Setup listener for exportToCSVDone
+    this.addEventListener('exportToCSVDone', () => {
+      this.exportingData = false;
```

Comment:
Currently, nothing is emitting the `exportToCSVDone` event

Reply:
Oh, I forgot to clean up that attempt that didn't work.  Calling the callback instead was cleaner and simpler.